### ISSUE TITLE
press and hold h, j, k, l

### ISSUE THREAD

**Issue by snoblenet** [onivim/oni2#1222]
In terminal Neovim, I am used to pressing and holding `h`, `j`, `k` or `l`, which creates smooth continual movement until I release the key. Could Onivim2 do the same?

**Comment by glennsl** [comment]
Hmm, I can't see any difference when comparing them myself. Can you illustrate with a gif or video?

**Comment by snoblenet** [comment]
Open file in Onivim2. Go to line 10. Press and hold k. The cursor moves to line 9, and stops.

Open file in Neovim in Alacritty. Go to line 10. Press and hold k. The cursor moves to line 9, then line 8, and so on, until you release the key.

**Comment by glennsl** [comment]
Are you on macOS Catalina? It might be a regression of https://github.com/revery-ui/revery/issues/703

**Comment by glennsl** [comment]
Relevant oni2 issue: https://github.com/onivim/oni2/issues/1149

**Comment by snoblenet** [comment]
Yes, on Catalina.

**Comment by glennsl** [comment]
Bummer :slightly_frowning_face: Sorry about that. I'll reopen the revery issue.

**Comment by CrossR** [comment]
Should be fixed with #1225 hopefully!

**Comment by snoblenet** [comment]
I've just downloaded the latest preview DMG and the issue remains FYI

**Comment by ericsolomon** [comment]
Repeated key movement is working as expected for me on Catalina 10.15.2 with build 0.3.0 (5d761658)

**Comment by snoblenet** [comment]
Has the DMG on the early access portal been updated? as it's still not working for me.

**Comment by harrisi** [comment]
TL;DR: run `defaults write -g ApplePressAndHoldEnabled -bool false` in a shell, log out and log back in.

There was a very annoying change for vim users in Catalina. Holding down keys changed behavior, and I'm still trying to entirely revert back, but for now, the solution to your problem is to run `defaults write -g ApplePressAndHoldEnabled -bool false` in a shell, log out and log back in ([source]). I just enabled it and tested in the most current build, and I get the unwanted behavior. Disabling it again fixes it.

In fact, the only reason it's not clear this is the issue is because, at least with my keyboard layout and internationalization settings, h, j, and k don't have alternate input options. Holding down l for a second or two will pop up a menu above the upper left corner of oni2's window, listing 'ł' as an input option. Pressing 1 will insert this (although there seems to be some utf-8 related bugs in oni2 so your input will be somewhat messed up after this).

Side note: It's interesting that the popup displays in the upper left corner of the oni2 window. In other editors (here, for example), it pops up immediately above the current input position.

[source]: https://apple.stackexchange.com/a/332770

**Comment by snoblenet** [comment]
that worked for me thanks
